packages/react/src/components/TreeView/TreeView.tsx

```diff
@@ -26,18 +78,21 @@ export default function TreeView({
   selected: preselected,
   size = 'sm',
   ...rest
-}) {
+}: TreeViewProps) => {
   const enableTreeviewControllable = useFeatureFlag(
     'enable-treeview-controllable'
   );
 
   const { current: treeId } = useRef(rest.id || uniqueId());
   const prefix = usePrefix();
   const treeClasses = classNames(className, `${prefix}--tree`, {
+    // @ts-ignore - will always be false according to prop types
     [`${prefix}--tree--${size}`]: size !== 'default',
   });
```

Comment:
I also found this line of code a bit strange, considering `size` should only be `xs` or `sm` (according to types).